### PR TITLE
Refactor crud.py

### PR DESCRIPTION
**Pull request by sameshl**
Related to #424 

### Checklist
- [x] My branch is up-to-date with upstream/develop branch.
- [x] Everything works and tested for Python 3.5.2 and above.

### Description

Refactor parts of `hydrus/data/curd.py`.

### CODE REVIEW

File: hydrus/data/crud.py [hunk 1]
```diff
@@ -254,11 +209,8 @@ def insert_multiple(objects_: List[Dict[str,
```
Comment (chrizandr) [inline hydrus/data/crud.py:209]: Can we not reuse insert here as is? What is preventing us to use insert in insert_mutliple? All these checks are already in insert, maybe we should reuse that function?

File: hydrus/data/crud.py [hunk 2]
```diff
@@ -410,30 +346,18 @@ def delete_multiple(
...
-        data_III += session.query(triples).filter(
-            triples.GraphIII.subject == index).all()
+        instance = get_instance_before_delete(session, index, type_)
+        instances.append(instance)
+        data_IAC_index, data_III_index, data_IIT_index = get_data_iac_iii_iit(session, index)
+        data_IIT += data_IAC_index
+        data_IAC += data_IAC_index
+        data_III += data_III_index
 
     data = data_III + data_IIT + data_IAC
```
Comment (chrizandr) [inline hydrus/data/crud.py:362]: Same here, these checks are already in delete. Maybe we can reuse the function with a few modifications.

File: hydrus/data/crud.py (hunk 1 above)
Comment (sameshl) [reply hydrus/data/crud.py:209]: Yes, makes sense. I will make this change

File: hydrus/data/crud.py (hunk 2 above)
Comment (sameshl) [reply hydrus/data/crud.py:362]: Yes, I will do this

**Comment by sameshl** [comment]
@chrizandr Made the requested changes

**Comment by xadahiya** [comment]
@chrizandr Please confirm the necessary changes are done, then we can merge

**Pull request merged**